py-data.csv

```diff
@@ -210,8 +210,8 @@ https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_contains,OD-Vic,,,
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_edit,OD-Vic,,,
 https://github.com/daknuett/ljson,1d3dc13001d9d2f61bcfbf9d5c5b16d44fb8076e,test/test_ljson_mem.py::test_unique_check,NIO,Accepted,https://github.com/daknuett/ljson/pull/4,
-https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,tests/test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_on_next_drop_new_message_when_buffer_full,OD-Vic,,,
-https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_when_on_next_buffer_following_messages,OD,,,https://github.com/daliclass/rxpy-backpressure/issues/4
+https://github.com/daliclass/rxpy-backpressure,b1a7bddfb370884135d6f2e7d9a3dd332a90d7cc,tests/test_sized_buffer_backpressure_strategy.py::TestDropBackPressureStrategy::test_on_next_drop_new_message_when_buffer_full,OD-Vic,RepoArchived,,
```

Comment:
I don't think this repo is archived?

Reply:
It was at the time of the commit, I think they have unarchived it after. I will update this.